llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -7801,6 +7600,13 @@ VectorizationFactor LoopVectorizationPlanner::computeBestVF() {
 
       InstructionCost Cost = cost(*P, VF);
       VectorizationFactor CurrentFactor(VF, Cost, ScalarCost);
+
+      if (RU.exceedsMaxNumRegs(TTI)) {
+        LLVM_DEBUG(dbgs() << "LV(REG): Ignoring VF " << VF
+                          << " as it uses too many registers\n");
```

Comment:
Nit, reword this like `"Not considering vector loop of width " << VF << " because it uses too many registers"` to be inline with the previous debug messages?